### ISSUE TITLE
systemd install tool/script?

### ISSUE THREAD

**Issue by karlicoss** [karlicoss/promnesia#15]
I've had to write `check_call(['systemctl', ...])` and use templating for systemd services more than I wish I had.
Can it be resolved in a more generic way?
e.g. [here](https://github.com/karlicoss/promnesia/blob/197af911eb0971a406a4f9edb68c2bc801870b7f/setup_server#L9) or [here](https://github.com/karlicoss/grasp/blob/c5440b61cb4faf85f843bc863bc46993ce1e61ef/server/setup#L9)

**Comment by karlicoss** [comment]
Some relevant stuff here https://github.com/karlicoss/grasp/issues/32

**Comment by albertz** [comment]
Note: I think such a thing (installing some user service) is generally useful. I also have some script ([this](https://github.com/albertz/mouse-scroll-wheel-acceleration-userspace)) where I would want to have such a systemd install script (or launchctl on MacOSX, and no idea what Windows has, or other Unixes).

Maybe it makes sense if we package sth like this as a separate PIP package, so that we can easily make use of it? (Or maybe there is already some project/package like this?)

**Comment by karlicoss** [comment]
Yeah, perhaps that would make sense, I've got some other projects that bodge this (including grasp, yeah). I'll do one more bout of search just to absolutely make sure nothing like this exists, but if not, we could do it!
